BoxContentSDK/BoxContentSDK/Requests/BOXFileCollaborationsRequest.m

```diff
@@ -43,7 +43,7 @@ - (BOXAPIOperation *)createOperation
     }
     
     if (self.nextMarker != nil) {
-        queryParameters[BOXAPIParameterKeyMarker] = self.nextMarker;
+        queryParameters[BOXAPIParameterKeyNextMarker] = self.nextMarker;
```

Comment:
because this line
https://github.com/box/box-ios-sdk/blob/objective-c-maintenance/BoxContentSDK/BoxContentSDK/BOXContentSDKConstants.h#L248

mark `BOXAPIParameterKeyMarker` as deprecated.

Reply:
I need to look into this one further...

Reply:
@OneSecure Sorry for the delay, this one was a little confusing.  We should not make this change.  It appears that deprecating `marker` and switching to `next_marker` was a bug.  

I just created an Issue for it here: https://github.com/box/box-ios-sdk/issues/708  It's a minor bug, so we probably won't be able to get it fixed in the near future.  However, if you would like to submit a PR to fix it in the meantime, we will definitely review it and get it merged!

Reply:
As your wish. changed it back.